lib/screens/app/profile/profile.dart

```diff
@@ -134,31 +130,42 @@ class _ProfileState extends State<Profile> {
                                 size: 16.0,
                                 color: Colors.black,
                               ),
-                              onPressed: () =>
-                                  _editProfilePicBottomSheet(context),
+                              onPressed: () => _editProfilePicBottomSheet(context),
                             ),
                           ),
                         )
                       ],
                     ),
                     Padding(
                       padding: EdgeInsets.only(top: 5),
-                      child: Text(
-                        SettingsNotifier.of(context).accountName,
-                        style: TextStyle(
-                          fontFamily: "worksans",
-                          fontSize: 24,
-                          color: Colors.white,
-                          fontWeight: FontWeight.w600,
-                        ),
+                      child: Column(
```

Comment:
this is the only real change here. Everything else is formatting. 

Note: I am using android studio formatting standards.

Reply:
https://flutter.dev/docs/development/tools/formatting